doc/LIMITS.md

```diff
@@ -106,87 +136,90 @@ libp2p stream multiplexers impose limits on the amount of streams that can be op
 
 These settings are done on a per-muxer basis, please see the README of the relevant muxer you are using.
 
+All fields are optional. The default values are defined in [@libp2p/mplex/src/mplex.ts](https://github.com/libp2p/js-libp2p-mplex/blob/master/src/mplex.ts) - please see that file for the current values.
+
 ### Mplex
 
 [@libp2p/mplex](https://github.com/libp2p/js-libp2p-mplex) supports the following:
 
-```js
+```ts
 const node = await createLibp2pNode({
   muxers: [
-    new Mplex({
+    mplex({
       /**
        * The total number of inbound protocol streams that can be opened on a given connection
        */
-      maxInboundStreams: 1024,
+      maxInboundStreams: number
 
       /**
        * The total number of outbound protocol streams that can be opened on a given connection
        */
-      maxOutboundStreams: 1024,
+      maxOutboundStreams: number
 
       /**
-       * How much incoming data to buffer before resetting the stream
+       * How much incoming data to buffer while attempting to parse messages - peers sending many small messages in batches may cause this buffer to grow
```

Comment:
Lets be clear about the units here.  Are we limiting the *number* of unprocessed messages in the queue, or limiting the number of *bytes* of unprocessed messages?

Reply:
The `maxUnprocessedMessageQueueSize` is the incoming raw data that's yet to be processed into messages so it's bytes.  I've updated the comment but am also open to better names!

Reply:
Got it.  I think a clearer name is ``maxUnprocessedMessageQueueBytes``.